db/schema.rb

```diff
@@ -10,194 +10,187 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170330225217) do
+ActiveRecord::Schema.define(version: 20170610162757) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
   enable_extension "hstore"
 
-  create_table "activities", force: :cascade do |t|
-    t.integer  "team_id"
-    t.string   "kind",         limit: 255
-    t.string   "guid",         limit: 255
-    t.string   "author",       limit: 255
-    t.string   "title",        limit: 255
-    t.text     "content"
-    t.string   "source_url",   limit: 255
+  create_table "activities", id: :serial, force: :cascade do |t|
```

Comment:
Yes, there have been changes to the `create_table` method, such as that the definition of primary keys is now part of this method. The Postgres adapter then uses [this function](https://github.com/rails/rails/blob/c6b4b4a52c7ec66d1923ef44dcbc71b582d7b165/activerecord/lib/active_record/connection_adapters/postgresql/schema_definitions.rb#L44) to define it *(it could for instance also be a UUID or a big-serial)*.